thanos/charts/values.yaml

```diff
@@ -26,6 +26,8 @@ thanos:
   # @default -- 0.0.0.0:10902
   httpAddress:
 
+  objectstoreSecretName:
+
```

Comment:
The secret name is used for both the compactor and the store, so this value should be valid for both.